package.json

```diff
@@ -1,7 +1,7 @@
 {
   "author": "EverydayHero <edh-dev@everydayhero.com>",
   "name": "edh-widgets",
-  "version": "0.2.5",
+  "version": "0.2.6",
```

Comment:
We will do that by the end of the day? I would like to have my story finished before demo tomorrow.